### PR TITLE
*: cut 0.6.1

### PR DESCRIPTION
**Pull request by brancz**
@stuartnelson3 @fabxc @brian-brazil @grobie @mxinden 

/cc @prymitive @mattbostock

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## v0.6.1 / 2017-04-28
+
+- [BUGFIX] Fix broken OpsGenie integration
```
Comment (fabxc) [inline CHANGELOG.md]: The fix was more general for all integrations using a hash for deduplication. Just that OpsGenie is the only reported one so far.

Comment (brancz) [inline CHANGELOG.md]: Yes I realize that, just wasn't sure how to phrase it to be meaningful from a user's perspective. Wdyt of `Fix incorrectly handled hash for notification providers`?

Comment (fabxc) [inline CHANGELOG.md]: SGTM

Comment (brancz) [inline CHANGELOG.md]: done

**Comment by prymitive** [comment]
I'm a bit too late to the party but there's one tiny glitch to fix:

```
"alerts": [
          {
            "labels": {
              "alertname": "TestCAlert9",
              "cluster": "prod",
              "severity": "critical"
            },
            "annotations": null,
            "startsAt": "2017-04-28T11:25:02.877648848-07:00",
            "endsAt": "0001-01-01T00:00:00Z",
            "generatorURL": "",
            "Status": "active",
            "inhibitedBy": [],
            "silencedBy": []
          }
        ]
```

Notice the uppercase on `Status`

**Pull request merged**